Doc/library/importlib.rst

```diff
@@ -435,8 +435,9 @@ ABC hierarchy::
             package. This attribute is not set on modules.
 
         - :attr:`__package__`
-            The parent package for the module/package. If the module is
-            top-level then it has a value of the empty string. The
+            The fully-qualified name of the package under which the module was
+            loaded as a submodule (or the empty string for top-level modules).
```

Comment:
```suggestion
            loaded as a submodule.
```

Reply:
I think the comment in the original PR is valuable here though.  This section is talking about the legacy API so having to "fast forward" in the documentation (since no link to the below section is provided) is a bit inconvenient.  The original text of the PR seems fine to me.